zenodo/modules/spam/views.py

```diff
@@ -121,6 +122,32 @@ def delete(user_id):
         return render_template('zenodo_spam/delete.html', **ctx)
 
 
+def _parse_email(email):
+    try:
+        prefix, domain = email.split('@', 1)
+        prefix = prefix.lower().strip()
+        domain = domain.lower().strip()
+        if domain[-1] == ".":
+            domain = domain[:-1]
+        return prefix, domain
+    except ValueError:
+        return None, None
```

Comment:
Can you check if this kind of logic is used in other places as well, and re-use this function? I think I've seen this before :)

Reply:
I got it from one of Lars' scripts, but we are using a one-liner elsewhere, so I swapped it :)